src/lib/logical_query_plan/drop_view_node.cpp

```diff
@@ -28,6 +32,15 @@ std::string DropViewNode::description() const {
 
 bool DropViewNode::subtree_is_read_only() const { return false; }
 
+const std::vector<std::string>& DropViewNode::output_column_names() const { return _output_column_names_dummy; }
```

Comment:
I don’t know when this method is used, but would it make sense to call `Fail()` instead?

Reply:
Nope, needs an actual implementation for the comparison.